core/templates/dev/head/collection_editor/CollectionDetailsDirective.js

```diff
@@ -0,0 +1,93 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive displaying and editing a collection details. Edit
```

Comment:
Also, is this meant to be an editor or a viewer? If the former, maybe call it CollectionDetailsEditorDirective? If the latter, that's a bit confusing, and I don't see an isEditable option...